### PR TITLE
update banner message

### PR DESCRIPTION
**Pull request by diriquelme**
## What does this PR do?
“For designs launching BEFORE July 14, 2021, please reference this (https://v13--spark-design-system.netlify.app) version of Spark. Talk to your PO or Experience Director if there are any questions.”

### Associated Issue
#3651 

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR then please remove it.

### Documentation
 - [ ] Update Spark Docs

### Browser Testing (current version and 1 prior)
  - [ ] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)
  
### Accessibility Testing
  - [ ] Axe browser extension
  - [ ] Jaws Inspect
  - [ ] VoiceOver (iOS) or Talkback (Android)

### Deploy Preview Reviewed and Approved
 - [ ] Design Team
 - [ ] Accessibility Team

### Screenshots
main_update_banner_message
Screen Shot 2020-11-17 at 8 21 55 AM

### CODE REVIEW

**Review by afebbraro — CHANGES_REQUESTED** [review]
Hey @diriquelme thanks for the PR. I requested changes because the whole sentence should not be a link. Can you fix that?

**Comment by diriquelme** [comment]
Hey @afebbraro made suggested changes and ready for review.

Update_11 18

File: src/components/layouts/Layout.js [hunk 1]
```diff
@@ -76,17 +76,21 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
-              View the previous major release of Spark
-            </a>
+          <div className="sprk-u-BackgroundColor--black sprk-u-AbsoluteCenter sprk-u-pam">
+            <p className="sprk-u-Color--white">
+              <span className="sprk-u-FontWeight--bold sprk-u-mrs">
+                Designs launching before July 14, 2021
+              </span>
+              should reference
+              <a
+                href="https://sparkdesignsystem.com/"
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:86]: Hey @diriquelme  this is the wrong link. It takes them back to where they are. You want the v13 link here.

**Review by afebbraro — CHANGES_REQUESTED** [review]
Requested some changes.

File: src/components/layouts/Layout.js (hunk 1 above)
Comment (diriquelme) [reply src/components/layouts/Layout.js:86]: Link has been updated to point to v13 https://v13--spark-design-system.netlify.app/

**Comment by diriquelme** [comment]
@afebbraro - Banner has been updated to match content approved formatting - https://deploy-preview-3660--spark-design-system.netlify.app/ Thank you for your feedback and comments :)

File: src/components/layouts/Layout.js [hunk 2]
```diff
@@ -76,17 +76,20 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
...
-              href="https://v13--spark-design-system.netlify.app/"
-              className="docs-c-Banner--link sprk-u-mlm"
-            >
-              View the previous major release of Spark
-            </a>
+          <div className="sprk-u-BackgroundColor--black sprk-u-AbsoluteCenter sprk-u-pam">
+            <p className="sprk-u-Color--white">
+              Designs launching
+              <span className="sprk-u-FontWeight--bold sprk-u-mas">before</span>
+              July 14, 2021, please reference
```
Comment (afebbraro) [inline src/components/layouts/Layout.js:83]: Hey @diriquelme , the year should be 2020.

**Review by afebbraro — CHANGES_REQUESTED** [review]
Everything looks good besides the year. It says 2021 and should be 2020. I wish it was 2021 though haha.

**Review by afebbraro — APPROVED** [review]

**Pull request merged**